app/code/Magento/GraphqlPlayground/view/frontend/templates/playground.phtml

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+// @codingStandardsIgnoreFile
+
+/** @var string $graphqlEndpoint Magento default Graphql Endpoint */
+$graphqlEndpoint =  $block->getBaseUrl() . "graphql";
```

Comment:
`graphql` frontname can be changed and should not be hardcoded here. See https://github.com/magento/magento2/blob/2.3/app/code/Magento/GraphQl/etc/di.xml#L21